### PR TITLE
Performance settings section translation

### PR DESCRIPTION
**Pull request by Withers001-1**
Added the translation for the new settings performance-related in the Alpha, and made part of the file look cleaner.

### CODE REVIEW

File: translations/base-fr.yaml [hunk 1]
```diff
@@ -722,11 +723,13 @@ settings:
                 fast: Rapide
                 super_fast: Très rapide
                 extremely_fast: Extrêmement rapide
+
```
Comment (DerFeldspatz) [inline translations/base-fr.yaml:726]: You need to remove all the spaces in a free line. This also applies to lines 732, 749, 754, 763, 769, 775, 780, 785 and 790. You can look this up from the error messages in the CI check

File: translations/base-fr.yaml [hunk 2]
```diff
@@ -743,10 +746,12 @@ settings:
                 ten_minutes: 10 Minutes
                 twenty_minutes: 20 Minutes
                 disabled: Désactivé
-        compactBuildingInfo:
+       
+       compactBuildingInfo:
```
Comment (DerFeldspatz) [inline translations/base-fr.yaml:750]: There is one space missing here in front (for a total of 8).

**Review by DerFeldspatz — CHANGES_REQUESTED** [review]

File: translations/base-fr.yaml [hunk 3]
```diff
@@ -755,11 +760,39 @@ settings:
...
         rotationByBuilding:
             title: Rotation par catégorie de bâtiment
             description: >-
                 Chaque catégorie de bâtiment enregistre le sens de rotation que vous lui avez assigné la dernière fois, de manière individuelle.
                 Cela sera sans doute plus confortable si vous alternez fréquemment entre le placement de différents types de bâtiments.
+        
+        lowQualityMapResources:
+            title: Ressources de la carte de plus basse qualité
+            description: >-
+                Simplifie le rendu des ressources sur la carte lorsqu'elle est zoomée opur améliorer les performances.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:773]: typo: `opur` -> `pour`

File: translations/base-fr.yaml [hunk 4]
```diff
@@ -755,11 +760,39 @@ settings:
...
+       
+        disableTileGrid:
+            title: Desactiver la grille de placement
+            description: >-
+                Desactiver la grille de placement peut aider les performances. Ça rend aussi le jeu encore plus uni!
+        
+        clearCursorOnDeleteWhilePlacing:
+            title: Effacer le curseur avec clic droit
+            description: >-
+                Activé par défaut, efface le curseur lorsque vous faites un clic droit en ayant un bâtiment selectioné pour la constructio. Si desactivé, vous pouvez detruire les bâtiments en faisant un clic droit tout en placant un bâtiment.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:784]: typo: `constructio` -> `construction`

File: translations/base-fr.yaml [hunk 5]
```diff
@@ -755,11 +760,39 @@ settings:
...
+        
+        lowQualityTextures:
+            title: Textures de basse résolution (Moche)
+            description: >-
+                Utilise des textures de basse qualité pour augmenter les performances. Cela va rendre le jeu moche!
+        
+        displayChunkBorders:
+            title: Monter les bordures de chunks
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:792]: Inconsistent choice of vocabulary here

File: translations/base-fr.yaml [hunk 6]
```diff
@@ -755,11 +760,39 @@ settings:
...
                 Cela sera sans doute plus confortable si vous alternez fréquemment entre le placement de différents types de bâtiments.
+        
+        lowQualityMapResources:
+            title: Ressources de la carte de plus basse qualité
+            description: >-
+                Simplifie le rendu des ressources sur la carte lorsqu'elle est zoomée opur améliorer les performances.
+                C'est encore plus clean, n'oubliez pas d'essayer !
+       
+        disableTileGrid:
+            title: Desactiver la grille de placement
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:777]: Typo: `Desactiver` -> `Désactiver` (throughout)

File: translations/base-fr.yaml [hunk 7]
```diff
@@ -755,11 +760,39 @@ settings:
...
+            description: >-
+                Desactiver la grille de placement peut aider les performances. Ça rend aussi le jeu encore plus uni!
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:779]: Kinda unclear

**Comment by tobspr** [comment]
I'll merge it for now, feel free to fix it with another pr @LeopoldTal ! :)

**Pull request merged**